core/server/src/main/java/alluxio/worker/file/FileSystemWorker.java

```diff
@@ -98,4 +114,44 @@ public void stop() {
     mFileSystemMasterWorkerClient.close();
     getExecutorService().shutdown();
   }
+
+  /**
+   * Cancels a file currently being written to the under file system. The open stream will be
+   * closed and the partial file will be cleaned up.
+   *
+   * @param workerFileId the id of the file to cancel, only understood by the worker that created
+   *                     the file
+   * @throws FileDoesNotExistException if this worker is not writing the specified file
+   * @throws IOException if an error occurs interacting with the under file system
+   */
+  public void ufsCancelFile(long workerFileId) throws FileDoesNotExistException, IOException {
```

Comment:
is it true that this file id is local to the worker and only meaningful for ufs related operations? If that is the case, how about `ufsFileId` instead of `workerFileId`? I feel `workerFileId` is a bit confusing